### PR TITLE
Bhathoot

### PR DESCRIPTION
**Pull request by Bhathoot**

### CODE REVIEW

**Comment by Bhathoot** [comment]
Benjamin Hathoot 47552457

File: 07-javascript.js [hunk 1]
```diff
@@ -143,6 +168,20 @@ assert(p.hasOwnProperty("name"))
 // Penalty: -3 layout, -3 naming
 
 //START
+function bugs(whatsUp){
+  backup = String.prototype.sup
```
Comment (pragdave) [inline 07-javascript.js:172]: Should declare this with `let backup = ...`

File: 07-javascript.js [hunk 2]
```diff
@@ -193,6 +232,9 @@ assert.equal("DOC".sup(), "<sup>DOC</sup>")
 
 function myNew(constructor, ...args) {
   //START
+let obj = Object.create(constructor.prototype)
```
Comment (pragdave) [inline 07-javascript.js:235]: Cool using `apply` and not `call`.

# -1 for no indent, though...

File: 07-javascript.js [hunk 3]
```diff
@@ -66,7 +66,12 @@ let str3 = "beef"
 let str4 = "briefly"
 let str5 = "Offensive Word"
 
+//Code sourced from: https://stackoverflow.com/questions/13829289/javascript-strip-vowels
 // START
+String.prototype.bowlderize = function(){
+let newStr = this.replace(/[aeiou]/ig,'*')
+return newStr
+}
```
Comment (pragdave) [inline 07-javascript.js:74]: Please indent your code

# -1

**Comment by pragdave** [comment]
# 78 / 80

**Pull request closed without merging**